x-pack/plugins/task_manager/server/routes/health.test.ts

```diff
@@ -662,6 +662,54 @@ describe('healthRoute', () => {
       },
     });
   });
+
+  it('returns a OK status for empty if shouldRunTasks is false', async () => {
+    const router = httpServiceMock.createRouter();
+
+    const stats$ = new Subject<MonitoringStats>();
+    const { serviceStatus$ } = healthRoute({
+      router,
+      monitoringStats$: stats$,
+      logger,
+      taskManagerId: uuidv4(),
+      config: getTaskManagerConfig({
+        monitored_stats_required_freshness: 1000,
+        monitored_aggregated_stats_refresh_rate: 60000,
+      }),
+      kibanaVersion: '8.0',
+      kibanaIndexName: '.kibana',
+      getClusterClient: () => Promise.resolve(elasticsearchServiceMock.createClusterClient()),
+      usageCounter: mockUsageCounter,
+      shouldRunTasks: false,
+      docLinks,
+    });
+    const serviceStatus = getLatest(serviceStatus$);
```

Comment:
NIT I know it's not on the scope of this PR, but official way to get first emission exists [firstValueFrom](https://rxjs.dev/api/index/function/firstValueFrom)

Reply:
Updated in https://github.com/elastic/kibana/pull/177631/commits/e46e745c38fdeb45011df7b2e5c2c09b07505123